doc/specs/Configuration object for profiles.md

```diff
@@ -0,0 +1,125 @@
+---
+author: <Pankaj> <Bhojwani> <pabhojwa@microsoft.com>
+created on: <2020-11-20>
+last updated: <2020-12-21>
+issue id: <#8345>

```

Comment:
you don't need the `<>` on these -- they're in the template to indicate that they're a whole field that you should replace. :smile:

Reply:
Oh oops, changed